python/cudf/cudf/tests/pytest.ini

```diff
@@ -4,3 +4,7 @@
 markers =
     spilling: mark benchmark a good candidate to run with `CUDF_SPILL=ON`
 xfail_strict = true
+filterwarnings =
+    error
+    ignore:::.*xdist.*
+    ignore:::.*pytest.*
```

Comment:
I ran into this as well. Specifying a module regex isn't actually supported; you'll need to specify the fully qualified module path.

I had to do something like this https://github.com/pandas-dev/pandas/pull/50386/files#diff-fad2fa83c042cce646b2c1a0455e9b60c22dd5f105ed8859ce85dcf361c07fbdR34

Reply:
Here's a set of related links about warnings and modules (I contributed a review on one of the CPython PRs below in hopes to get the docs to be aligned with the implementation):

https://stackoverflow.com/questions/71112211/why-does-pythons-filterwarning-module-regex-not-work-as-expected
https://github.com/python/cpython/pull/23172
https://github.com/python/cpython/pull/9358

Reply:
Specifying a module regex is only unsupported when passed from the command line. It works fine when used in Python code, or in this case when specified to pytest via an ini file. Quoting from [the relevant section of the warnings documentation](https://docs.python.org/3/library/warnings.html#the-warnings-filter):
> module is a string containing a regular expression that the start of the fully qualified module name must match, case-sensitively. In -W and PYTHONWARNINGS, module is a literal string that the fully qualified module name must be equal to (case-sensitively), ignoring any whitespace at the start or end of module. I believe there's a CPython bug somewhere discussing this inconsistency (and it wasn't always documented accurately either).

Reply:
Ah Bradley's comment didn't refresh onto my page until after I submitted. The cpython issues are the ones I was thinking of though!

Reply:
Ah! Okay that makes sense why my attempts on the pandas side failed before. Thanks for the heads up!